src/setupTests.js

```diff
@@ -1,4 +1,77 @@
+import React from 'react'
+import { act } from 'react-dom/test-utils'
 import Adapter from 'enzyme-adapter-react-16'
-import { configure } from 'enzyme'
+import { configure, mount } from 'enzyme'
+import 'jest-localstorage-mock'
+
+import App from './App.js'
+import { handleKeyboard } from './shortcuts.js'
 
 configure({ adapter: new Adapter() })
+
+const createTestApp = async (app) => {
+  document.getSelection = () => ({
+    type: 'None',
+    removeAllRanges: () => {},
+  })
+
+  document.createRange = () => ({
+    setStart: () => {},
+    setEnd: () => {},
+    commonAncestorContainer: {
+      nodeName: 'BODY',
+      ownerDocument: document,
+    },
+    collapse: () => {},
+  })
+
+  window.getSelection = () => ({
+    focusOffset: 3,
+    removeAllRanges: () => {},
+    addRange: () => {},
+  })
+
+  window.location = {
+    pathname: '',
+  }
+
+  await act(async () => {
+    // eslint-disable-next-line no-undef
```

Comment:
I believe you can remove the eslint-disable lines related to jest now that we have `jest` specified as an eslint env.